CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 # dev
 
+* Enhancement: Unified logging on stdout/stderr. Added option `--silent`. Printed text is colored only when output is a terminal ([#791](https://github.com/avast/retdec/issues/791).
```

Comment:
As @s3rvac suggested in some other place, it is better to add these entries after the merge to `master`. It is then possible to reference the PR.

Reply:
+ the risk of running into conflicts is much lower :slightly_smiling_face: